xds/src/main/java/io/grpc/xds/EnvoyServerProtoData.java

```diff
@@ -142,11 +142,11 @@ public int hashCode() {
     }
   }
 
-  static final class CidrRange {
+  public static final class CidrRange {
```

Comment:
These visibility changes wouldn't be necessary if `FilterChainMatchingHandler` were in this package, I suppose.

Reply:
Exactly.